main.py

```diff
@@ -1,10 +1,45 @@
+from django.db.models import ForeignKey  # noqa: F401
 import init_django_orm  # noqa: F401
-
+import json
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json") as players_file:
+        players = json.load(players_file)
+
+    for name, value in players.items():
+
+        race, _ = Race.objects.get_or_create(
+            name=value["race"]["name"],
+            description=value["race"]["description"]
+        )
+
+        guild = None
+        if value["guild"] is not None:
+            guild, _ = Guild.objects.get_or_create(
+                name=value["guild"]["name"],
+                description=(
+                    value["guild"]["description"]
+                    if value["guild"]["description"]
+                    else None
+                ),
+            )
+
+        for skill in value["race"]["skills"]:
+            Skill.objects.get_or_create(
+                name=skill["name"],
+                bonus=skill["bonus"],
```

Comment:
Ensure that the `bonus` value from the JSON matches the data type expected by the `Skill` model. If the `bonus` field in the model is changed to a numerical type (as suggested in the models.py review), make sure to convert or validate the `bonus` value accordingly before saving it to the database.